File: 241122/숫자 사이의 배수/multiplication-between-numbers.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.

    int a,b, sum_val=0, cnt=0;

    scanf("%d %d", &a, &b);

    for (int i=a; i<=b; i++){
        if (i%5==0 || i%7==0){
            sum_val+=i;
            cnt++;
        }
    }
    printf("%d %.1lf", sum_val, (double)sum_val/cnt);
    return 0;
}

File: 241127/숫자 사이의 배수/multiplication-between-numbers.py
arr = list (map(int, input().split()))

a = arr[0]
b = arr[1]

sum_val = 0
cnt = 0

for i in range(a,b+1) :
    if (i%5==0 or i%7==0) :
        cnt+=1
        sum_val+=i

print(f"{sum_val} {sum_val/cnt:.1f}")